plugins/k8sclient/options_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8sclient_test
+
+import (
+	"testing"
+
+	"github.com/ligato/networkservicemesh/plugins/k8sclient"
+	"github.com/ligato/networkservicemesh/utils/helper/deptools"
+	"github.com/ligato/networkservicemesh/utils/registry/testsuites"
+	. "github.com/onsi/gomega"
+)
+
+func TestSharedPlugin(t *testing.T) {
```

Comment:
I think TestSharedPlugin, TestNewPluginNotShared, and TestSharedPluginRemoveonClose should now be covered by TestWithRegistry :)

Reply:
In fact you appear to be correct, I can remove these tests. Thanks!